modules/virt-edit-cdrom-vm.adoc

```diff
@@ -19,27 +19,21 @@ endif::[]
 [id="virt-vm-edit-cdrom_{context}"]
 
 = Editing CD-ROMs for {object-gui}
-Use the following procedure to configure CD-ROMs for virtual machines.
+Use the following procedure to edit CD-ROMs for {object}s.
 
 .Procedure
 
-. From the *{object-gui}* tab, select your {object}.
+. Click *Workloads* -> *Virtualization* from the side menu.
 
-. Select the *Overview* tab.
+. Click the *{object-gui}* tab.
 
```

Comment:
@sjhala-ccs , @aburdenthehand Hi _ I don't have access today to a server build :-(  

The VM Overview screen belongs only the flow for VMs. 
Can a conditional expression be created such that a step shows up in Procedure A (for VMs) and not for Procedure B (for VM templates). If that is possible, then the additional step can be added such that it shows up only in the procedure for the VMs. Please advise.

Reply:
@lmandavi Like I said in my original comment, you can add the "VM Overview screen" part of the sentence if you declare another conditional. For example, set `object-overview` to `Virtual Machine Overview` for the VM context and the same conditional to `Template Details` for the template context. Does that make sense?

Reply:
Let me know if I am missing out on something here, but here's what I am noting.

When you click on Virtual Machines, the VM Overview screen displays (step 1)

Click on Details, and the VM Details screen displays (this is step 2)

(So that's 2 steps in the flow to get to the VM Details).

When you click on Templates, the Template Details screen displays.

(So that's 1 step in the flow to get to the Template Details screen and in this flow we don't have the Overview step for Templates the way we have it for the VM flow).

In this scenario, how does having a conditional expression help when the flow for VM requires an additional step (from VM Overview, you click on VM Details) but the flow for VM Templates does not require that additional step?

Do let me know if I am missing something here!

Reply:
@lmandavi I don't have access to a test env but iirc, you do not need to go to the VM Details screen to access the Disks tab (which is the next step). The different tabs are accessible from the VM Overview screen itself.

Reply:
The famous {object-overview} conditional is now present in three .adoc files: add a nic, add a disk, and edit cd-rom.